coreos-base/oem-packet/files/units/packet-phone-home.service

```diff
@@ -1,13 +1,14 @@
 [Unit]
-Description=Basic setup tasks to initialize the machine.
+Description=Report Success to Packet
 ConditionFirstBoot=true
 Requires=coreos-metadata.service
 After=coreos-metadata.service
 
 [Service]
 EnvironmentFile=/run/metadata/coreos
-Type=oneshot
-ExecStart=/usr/bin/curl --request POST "${COREOS_PACKET_PHONE_HOME_URL}"
+ExecStart=/usr/bin/curl --header "Content-Type: application/json" --request POST "${COREOS_PACKET_PHONE_HOME_URL}"
+Restart=on-failure
```

Comment:
I think this will only fail and restart when it is unable to reach the server.  Do HTTP response codes matter?  If so, `curl` should have `--fail`.